pep600_compliance/images/os/centos.py

```diff
@@ -31,9 +31,9 @@ def install_packages(self, container, machine):
 
 CENTOS_LIST: list[base.Base] = [
     CentOS(
-        "quay.io/centos/centos:stream9-development",
-        "rolling",
-        machines=["x86_64", "aarch64"],  # TODO "ppc64le", "s390x"
+        "quay.io/centos/centos:stream9",
+        ("EOL:~2027-05-31",),
```

Comment:
```suggestion
        ("EOL:2027-05-31",),
```
I guess it's probably subject to change but an isoformat string is required here to allow for date comparison later on.